src/theory/arrays/theory_arrays.cpp

```diff
@@ -2053,14 +2047,13 @@ bool TheoryArrays::dischargeLemmas()
 
 void TheoryArrays::conflict(TNode a, TNode b) {
   Debug("pf::array") << "TheoryArrays::Conflict called" << std::endl;
-
-  explain(a.eqNode(b), d_conflictNode);
-
-  if (!d_inCheckModel) {
-    d_out->conflict(d_conflictNode);
+  if (d_inCheckModel)
+  {
+    // if in check model, don't send the conflict?
```

Comment:
why `?` ? :-)

Reply:
Haha, I still don't know why, but I guess the comment shouldn't have "?" :)